.azurepipelines/Platform-Build-GCC5.yml

```diff
@@ -56,7 +56,35 @@ jobs:
           **/QEMUQ35_*.fd
         BuildArtifactsOther: "**/unit_test_results/*"
 
-      QemuQ35_DEBUG_ARM:
+      QemuQ35x86_DEBUG:

```

Comment:
nit: naming here is a bit confusing, consider something like IA32X64 to follow OVMF to distinguish this from an entirely x86 (32 bit) build

Reply:
I also recommend sticking with "IA32" vs "x86" to keep consistent with the build architectural specifiers. I agree that "IA32X64" is the most obvious name to use.

Reply:
Updated.